Detections/AzureDiagnostics/KeyVaultSensitiveOperations.yaml

```diff
@@ -20,7 +20,7 @@ query: |
 
   let timeframe = 1d;
   let SensitiveOperationList = dynamic(
-  ["VaultDelete", "KeyDelete", "SecretDelete", "SecretPurge", "KeyPurge", "SecretBackup", "KeyBackup"]);
+  ["VaultDelete", "KeyDelete", "KeyDecrypt", "SecretDelete", "SecretPurge", "KeyPurge", "SecretBackup", "KeyBackup"]);
```

Comment:
Please reference PR #849 where `KeyDecrypt` was removed due to being noisy as compared with other operations. 

We may have to monitor `KeyDecrypt` with the combination of IP Allowlist.

Reply:
For the moment I've changed the commit to remove `KeyDecrypt` from the description.